### PR TITLE
Feature: Chromecast connector

### PR DESCRIPTION
**Pull request by snooky23**
This P.R made in order to connect ApplicasterSDK player to the Chromecast logic in Zapp-iOS.

### CODE REVIEW

File: ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//
+//  ZAAppDelegateConnectorChromecastProtocol.h
+//  ZappAppConnector
+//
+//  Created by Avi Levin on 21/12/2017.
+//  Copyright © 2017 Applicaster Ltd. All rights reserved.
+//
+@protocol ZAAppDelegateConnectorChromecastProtocol
```
Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Why isn't this in Swift?

Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Please add a space after `@protocol ...` and before `@end` to make it cleaner.

Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Please add documentation.

Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Please add nullability annotations (or just write it in Swift).

Comment (snooky23) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Because the connector code is already in OBJ C.

Comment (snooky23) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: You mean "enter" not "space". done :)

Comment (snooky23) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Done

Comment (snooky23) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: This protocol is used to connect between 2 obj c sides. 
I added NS_ASSUME_NONNULL_BEGIN and NS_ASSUME_NONNULL_END anyway.

Let talk more about it on Sunday/Monday.

Comment (PRuedaApplicaster) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: And what is the problem of using Swift for this protocol even if the class that is going to implement it is objective c?

**Pull request merged**